emission/analysis/modelling/similarity/similarity_metric.py

```diff
@@ -27,15 +27,21 @@ def similarity(self, a: List[float], b: List[float]) -> List[float]:
         """
         pass
 
-    def similar(self, a: List[float], b: List[float], thresh: float) -> bool:
+    def similar(self, a: List[float], b: List[float], thresh: float, clusteringWay :str = 'origin-destination') -> bool:
         """compares the features, returning true if they are similar
         within some threshold
 
         :param a: features for a trip
         :param b: features for another trip
         :param thresh: threshold for similarity
+        :clusterinWay: clustering based on origin/destination/origin-destination-pair
         :return: true if the feature similarity is within some threshold
         """
         similarity_values = self.similarity(a, b)
```

Comment:
`similarity_values` takes the form ['origin-origin' point distance, 'destination-destination' point distance] as returned from [`similarity`](https://github.com/e-mission/e-mission-server/blob/94e7478e627fa8c171323662f951c611c0993031/emission/analysis/modelling/similarity/od_similarity.py#L21) function.

Reply:
Ditto wrt comments

Reply:
These changes are now reverted.

Reply:
These changes are now reverted.